apis/v1alpha1/shared_types.go

```diff
@@ -51,27 +51,43 @@ type NamespacedPodSubject struct {
 // +kubebuilder:validation:MinProperties=1
 type AdminNetworkPolicyPort struct {
 	// Port selects a port on a pod(s) based on number.
+	//
+	// Support: Core
+	//
 	// +optional
 	PortNumber *Port `json:"portNumber,omitempty"`
 
 	// NamedPort selects a port on a pod(s) based on name.
+	//
+	// Support: Extended
```

Comment:
OK, apparently we still have "Extended". @tssurya ?

We should not have NamedPort support be optional. Either it's required, or it's out. (Note that if it's out then that makes it harder to write a generic "allow to DNS" policy.)

Reply:
yea I will remove the whole Extended support
Named ports can be graudated, we have enough implementations to call it stable at this point